django-stubs/forms/models.pyi

```diff
@@ -150,7 +150,7 @@ class BaseModelFormSet(Generic[_M, _ModelFormT], BaseFormSet[_ModelFormT]):
     def save_existing(self, form: _ModelFormT, instance: _M, commit: bool = ...) -> _M: ...
     def delete_existing(self, obj: _M, commit: bool = ...) -> None: ...
     saved_forms: List[_ModelFormT] = ...
-    save_m2m: Callable[[], None] = ...
+    save_m2m: Callable[[self], None] = ...
```

Comment:
Myabe we should re-write it as `def save_m2m`?

Reply:
ok, done.

Note that it is a method that will only be there when `commit=False` is used before, but I think that cannot be reflected really.